subsys/mpsl/mpsl_init.c

```diff
@@ -233,6 +226,19 @@ static int mpsl_signal_thread_init(const struct device *dev)
 	return 0;
 }
 
+#if IS_ENABLED(CONFIG_MPSL_FEM)
+static int mpsl_fem_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+	return mpsl_fem_configure();
+}
+#endif
+
 SYS_INIT(mpsl_lib_init, PRE_KERNEL_1, CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
 SYS_INIT(mpsl_signal_thread_init, POST_KERNEL,
 	 CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
+
+#if IS_ENABLED(CONFIG_MPSL_FEM)
+SYS_INIT(mpsl_fem_init, POST_KERNEL, CONFIG_MPSL_FEM_INIT_PRIO);
```

Comment:
Do we need it to be configurable? If the GPIO driver is at `CONFIG_KERNEL_INIT_PRIORITY_DEFAULT` we could just assign this to `CONFIG_KERNEL_INIT_PRIORITY_DEFAULT + 1`

Reply:
> we could just assign this to `CONFIG_KERNEL_INIT_PRIORITY_DEFAULT + 1`

This won't work. See [init.h](https://github.com/nrfconnect/sdk-zephyr/blob/master/include/init.h#L130-L133).

Reply:
Ok. What about using `CONFIG_KERNEL_INIT_PRIORITY_DEVICE` instead of creating a new kconfig?

Reply:
As far as I'm concerned this value doesn't need to be configurable. Using `CONFIG_KERNEL_INIT_PRIORITY_DEVICE` is a fine solution IMO